handler/openapi/model-cfg.yaml

```
@@ -0,0 +1,8 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/oapi-codegen/oapi-codegen/HEAD/configuration-schema.json
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)